circle.yml

```diff
@@ -512,6 +512,51 @@ jobs:
       - spectest
       - collect_coverage_data
 
+  bindings-rust:
+    docker:
+      - image: rust:1-buster
+    steps:
+      - checkout
+      - run:
+          name: Update environment
+          command: |
+            echo 'deb http://deb.debian.org/debian buster-backports main' >> /etc/apt/sources.list
+            apt -qq update
+            apt -yq install libclang-dev clang cmake --no-install-recommends
```

Comment:
Even though I add buster-backports, this still installs cmake 3.13 from buster, while buster-backports has 3.16: https://packages.debian.org/buster-backports/cmake

Reply:
Fixed.

Reply:
Why are you installing cmake twice? Also: alpine 3.12 has cmake 3.17.